code/src/layouts/intro-banner/IntroBanner.test.tsx

```diff
@@ -2,9 +2,20 @@ import { render, screen } from "@testing-library/react";
 import IntroBanner from "@/layouts/intro-banner/IntroBanner";
 
 describe("IntroBanner", () => {
-  test("should render IntroBanner title", () => {
+  beforeEach(() => {
     render(<IntroBanner />);
+  });
+
+  test("should render IntroBanner title", () => {
     const headerText = screen.getByText("introBanner.header");
     expect(headerText).toBeInTheDocument();
   });
+
+  test("should render IntroBanner badge with right properties", () => {
+    const badge =
+      screen.getByText("introBanner.badge").parentElement?.parentElement;
+
+    expect(badge).toBeInTheDocument();
+    expect(badge).toHaveClass("spa-badge__no-rounded");
```

Comment:
Thanks for suggestion, I added that test because of mutations. By default badge is rounded, so we need to pass `isRounded={false}` inside of `IntroBanner` like following:

```
<AppBadge
            variant="danger"
            size="large"
            isRounded={false}
            badgeContent={badgeContent}
          />
```

Mutation tests mutate it and sets true instead of false. So we need to check whether the badge is rounded, I guess the easiest way to do this is to check its class.